### PR TITLE
Rework of sass-lint config

### PR DESCRIPTION
**Pull request by koskinpark**

### CODE REVIEW

**Comment by Megajack** [comment]
Looks fine for me. Voting for merge

File: .sass-lint.yml [hunk 1]
```diff
@@ -93,11 +74,11 @@ rules:
   empty-args: 0
   hex-length: 2
   hex-notation: 2
-  # indentation: 2
+  indentation: 0
   leading-zero: 2
   nesting-depth:
     - 2
-    - max-depth: 4
+    - max-depth: 5
```
Comment (iberdinsky-skilld) [inline .sass-lint.yml]: why?

Comment (koskinpark) [reply .sass-lint.yml]: it's my personal experience. I increase it sometimes when need to theme multiple levels menu

Comment (iberdinsky-skilld) [reply .sass-lint.yml]: i would better change it to 3 levels actually. less chaos

**Comment by hog-seruj** [comment]
Let's check what we have in Drupal core now: https://www.drupal.org/node/2868114

**Comment by hog-seruj** [comment]
In D8.6 https://prettier.io/

**Comment by iberdinsky-skilld** [comment]
https://github.com/skilld-labs/kaizen/issues/3#issuecomment-465258374

**Comment by koskinpark** [comment]
> [#3 (comment)](https://github.com/skilld-labs/kaizen/issues/3#issuecomment-465258374)

https://github.com/skilld-labs/kaizen/pull/37/commits/2ba375fea49ee5941ab438bae842402ae3b4e4fc

**Comment by koskinpark** [comment]
pr is outdated

**Pull request closed without merging**